setup.py

```diff
@@ -2,6 +2,10 @@
 import sys
 from setuptools import setup
 from warnings import warn
+import versioneer
+
+version = versioneer.get_version()
```

Comment:
You're getting the version here, but not passing it to the `setup()`. From the `pandas` repo that you referenced, here is the [line](https://github.com/pandas-dev/pandas/blob/master/setup.py#L649) I'm referring to.